src/SwedbankPay.Sdk/PaymentInstruments/Card/CardPaymentRequest.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using SwedbankPay.Sdk.Common;
+using SwedbankPay.Sdk.PaymentOrders;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    public class CardPaymentRequest
+    {
+        public CardPaymentRequest(Operation operation,
+                              PaymentIntent intent,
+                              CurrencyCode currency,
+                              List<IPrice> prices,
+                              string description,
+                              string userAgent,
+                              CultureInfo language,
+                              IUrls urls,
+                              PayeeInfo payeeInfo,
+                              bool generatePaymentToken = false,
+                              bool generateReccurenceToken = false,
+                              string payerReference = null,
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)